schema.json

```diff
@@ -400,6 +472,11 @@
                 "type": "string"
               }
             },
+            "required": [
+              "network",
+              "username",
+              "url"
```

Comment:
Not sure we need to require a url. Maybe a network doesn't provide a profile page etc.

Reply:
I have thought about that too, but I could not find any example of a network without URL. Also, I think the URL is required for a theme to render correctly. Consider a theme like Orbit wants to render a less known network. Normally it renders the network icon near the profile username to specify which network is the username for. If the network name is unknown it needs to use someway to still show what is that profile. Rendering the URL alone would provide that information without breaking the UI.

Reply:
Fair point. We can reiterate with additional user feedback. Happy to keep it as is for now.